### PR TITLE
feat :  done with quiz answers

### PR DESCRIPTION
**Pull request by iprankurpandey**

### CODE REVIEW

**Comment by swastika0015** [comment]
I think the login button can be at the right corner and the result score could be in center with one line like "Hey, XYZ! Your score is XYZ".

**Comment by Sahil25061999** [comment]
Same thing as swastika suggested apart from that i think the shadows can be made a bit lighter

File: result.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+.first-brand{
+    display: flex;
+    height: 15rem;
+    width: 100%
+}
+
+.brand-logo {
+    width: 25rem;
+    /* min-height: 25rem; */
```
Comment (nikhiljugale007) [inline result.css:60]: you can remove commented code.

File: result.html [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            </div>
+        </div>
+        <!-- <div class="sixth-brand brand-logo"> -->
+        <!-- <img src="https://i.ibb.co/88Lb364/icons8-git-480.png" alt="css" class="img-conatiner"> -->
+        <!-- <div class="brand-about">
+                <div class="head-text"> CSS Quiz</div>
+                <p> CSS is a style sheet language used for describing the presentation of a document written in a markup
+                    language.
+                </p>
+                <button class="play-btn-quiz">PLAY</button> -->
```
Comment (nikhiljugale007) [inline result.html:157]: you should remove commented code.

**Review by nikhiljugale007 — COMMENTED** [review]
I am not getting why you have added button for each question in result page.
One more suggestion you can show which question is wrong and which right for that user.

**Review by Sahil25061999 — COMMENTED** [review]
Same thing as swastika suggested apart from that i think the shadows can be made a bit lighter

File: result.css [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    background-color: rgb(255, 255, 255);
+    border-radius: 10px;
+    display: flex;
+    justify-content: flex-start;
```
Comment (bhtibrewal) [inline result.css:64]: i dont think this will be needed

File: result.css [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    padding: 1rem;
+    box-shadow: 0px 0px 10px rgb(0 0 0 / 50%);
+    font-weight: bold;
+    color: rgb(0, 0, 0);
+    max-width: 100%;
+    height: auto;
+
+}
+.username,.score-text{
+    font-size: 1rem;
```
Comment (bhtibrewal) [inline result.css:77]: it should be by default 1rem

File: result.css [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+    margin: 0.5rem;
+}
+.option-btn-right{
+    background-color: green;
+    color: white;
+}
+.score-container{
+    /* margin: 1rem; */
+    /* padding: 1rem; */
+}
```
Comment (bhtibrewal) [inline result.css:148]: remove commented code

File: result.css [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+.search-panel {
+    display: flex;
+}
+
+.mtrlshop {
```
Comment (bhtibrewal) [inline result.css:153]: need to improve the naming

File: result.html [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+            <h3>Welcome : Tanay Pratap</h3>
+        </div>
+        
+            <div class="score-text">
+                <h3>Your Score : 08</h3>
+            </div>
+           
+    </div>
+    <div class="brand-choice">
+        <div class="first-brand brand-logo">
```
Comment (bhtibrewal) [inline result.html:59]: need to work on naming of classes, these are pretty misleading

File: result.html [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+
+            <!-- <img src="https://i.ibb.co/SRn0LwX/icons8-html-5-480.png" alt="html" class="img-conatiner"> -->
+
+            <div class="brand-about">
+                <div class="head-text"> Question - 1</div>
```
Comment (bhtibrewal) [inline result.html:64]: you could have used p/h3 tags here

File: result.html [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+                <div class="options">
+
+
+                  
+                    <button class="btn option-btn">Scripting Language</button>
+                    <button class="btn option-btn">Programming Language</button>
+                    <button class="btn option-btn-right"> Markup Language</button>
+                    <button class="btn option-btn">None</button>
+                </div>
+                <button class="play-btn-quiz">PLAY</button>
```
Comment (bhtibrewal) [inline result.html:93]: give `text-transform: uppercase` in play-btn-quiz class

File: result.html [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+
+    <footer class="footer-items">
+        <div class="h4"> Made with <code>
+                < />
+            </code>by Prankur Pandey </div>
+
+        <div class="social-media-links">
+            <div class="github"><i class="fa fa-github"></i></div>
+            <div class="twitter"><i class="fa fa-twitter"></i></div>
+            <div class="Linkedin"><i class="fa fa-linkedin"></i></div>
```
Comment (bhtibrewal) [inline result.html:170]: you can remove these extra divs and give the class directly to i

File: result.html [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+        </div>
+        <div class="H5">© 2021 Agri UI </div>
```
Comment (bhtibrewal) [inline result.html:172]: use p tag here

**Review by bhtibrewal — APPROVED** [review]
looks good to merge after some minor changes suggested

**Pull request merged**